### ISSUE TITLE
rstudio glyph error on linux terminal

### ISSUE THREAD

**Issue by kshauna** [rstudio/rstudio#4216]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.1.463
    OS Version      : Linux Ubuntu 18.04.1 LTS
    R Version       : 3.4.4 

### Steps to reproduce the problem
well the problem was 'reproduced' after I uninstalled rstudio and reinstalled it to see if that would fix the error. 
### Describe the problem in detail
I have recently started using Linux's Ubuntu 18.04.01 LTS and installed r-base + rstudio.
when I open rstudio from the Linux terminal, it opens, but also gives the following:

shauna@shauna-desktop:~$ rstudio
load glyph failed err=6 face=0xd378e0, glyph=2798
load glyph failed err=6 face=0xd378e0, glyph=2798

although it seems to work fine obviously there is an error given in the Linux terminal and I don't know how to address it. I tried removing and re-installing rstudio but I get the same error. Past questions or similar issues I found online ran the GNU GDB debugger command in the Linux terminal. I did also but eventually terminated it as it seemed to stall and maybe not fully execute. 

$gdb
GNU gdb (Ubuntu 8.1-0ubuntu3) 8.1.0.20180409-git
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word".
(gdb) file rstudio
Reading symbols from rstudio...(no debugging symbols found)...done.
(gdb) run
Starting program: /usr/bin/rstudio 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffe83e8700 (LWP 5986)]
[New Thread 0x7fffe7751700 (LWP 6019)]
[New Thread 0x7fffe6f50700 (LWP 6024)]
[New Thread 0x7fffe637a700 (LWP 6030)]
[New Thread 0x7fffe5b79700 (LWP 6031)]
[New Thread 0x7fff95424700 (LWP 6032)]
[New Thread 0x7fff87fff700 (LWP 6035)]
[New Thread 0x7fff877fe700 (LWP 6036)]
[New Thread 0x7fff86ffd700 (LWP 6037)]
[New Thread 0x7fff867fc700 (LWP 6038)]
load glyph failed err=6 face=0xd29f80, glyph=2798
load glyph failed err=6 face=0xd29f80, glyph=2798

^C
Thread 1 "rstudio" received signal SIGINT, Interrupt.
0x00007ffff2fd0bf9 in __GI___poll (fds=0xbf26b0, nfds=5, timeout=25) at ../sysdeps/unix/sysv/linux/poll.c:29
29	../sysdeps/unix/sysv/linux/poll.c: No such file or directory.
(gdb) Quit

I have no data previously analysed as I am just beginning this project. I have not found an explicit answer to correct this. I am very happy to get any help or to be directed to a previously solved forum that I must have missed. 

I will also include my rstudio Diagnostics Report. Danke! 

### Describe the behavior you expected
I simply expected rstudio to open without giving an error glyph in the Linux terminal. I don't trust rstudio completely now to continue using it until I fix this as I am concerned that it might crash at some point. 

[diagnostics-report.txt](https://github.com/rstudio/rstudio/files/2792494/diagnostics-report.txt)

**Comment by ronblum** [comment]
@kshauna Are you by chance running in a VM? This message, which is innocuous, can be caused by having video acceleration turned on. Turning it off will make the messages disappear, as described in #2258.

**Comment by kshauna** [comment]
@ronblum apologies for my delayed response and I will address the 'repro needed', still haven't grasped how to effectively ask questions here. 

I am not running in a VM. I have created a Linux Ubuntu dual-boot alongside Windows so I am running entirely via the Linux Ubuntu OS. I saw on other forums that addressed this that it didn't seem to be a major problem but it's annoying to see it appear each time and I am completely ignorant as to why.
